src/plugins/core/range.ts

```diff
@@ -284,6 +288,19 @@ export class RangeAdapter implements CommandHandler<CoreCommand> {
     });
   }
 
+  expandRange(range: Range, dimension: Dimension, quantity: number): Range {
```

Comment:
What about `extendRange` instead? (similar to the `extendTableDown` in the table plugin).
I originally though this would expand the range to take merges into account just like what `expandZone` does.